frontend/src/metabase/actions/containers/CreateActionForm/CreateActionForm.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { useCallback, useMemo } from "react";
+import { t } from "ttag";
+import * as Yup from "yup";
+import { connect } from "react-redux";
+
+import Button from "metabase/core/components/Button";
+import Form from "metabase/core/components/Form";
+import FormFooter from "metabase/core/components/FormFooter";
+import FormProvider from "metabase/core/components/FormProvider";
+import FormInput from "metabase/core/components/FormInput";
+import FormTextArea from "metabase/core/components/FormTextArea";
+import FormSubmitButton from "metabase/core/components/FormSubmitButton";
+import FormErrorMessage from "metabase/core/components/FormErrorMessage";
+
+import * as Errors from "metabase/core/utils/errors";
+
+import Actions, { CreateQueryActionOptions } from "metabase/entities/actions";
+
+import FormModelPicker from "metabase/models/containers/FormModelPicker";
+
+import type {
+  ActionFormSettings,
+  CardId,
+  WritebackQueryAction,
+} from "metabase-types/api";
+import type { State } from "metabase-types/store";
+import type Question from "metabase-lib/Question";
+
+const ACTION_SCHEMA = Yup.object({
+  name: Yup.string()
+    .required(Errors.required)
+    .max(100, Errors.maxLength)
+    .default(""),
```

Comment:
for some reason, the question class only lets us change displayName, which is why it's not getting pulled in here